src/commands/functions/functions-create.js

```diff
@@ -565,6 +624,28 @@ const installAddons = async function (command, functionAddons, fnPath) {
   return Promise.all(arr)
 }
 
+const registerEFInToml = async (funcName) => {
+  if (!fs.existsSync('netlify.toml')) {
+    log(`${NETLIFYDEVLOG} \`netlify.toml\` file does not exist yet. Creating it...`)
+  }
+
+  // Todo: validate this input
+  const { funcPath } = await inquirer.prompt([
+    {
+      type: 'input',
+      name: 'funcPath',
+      message: `What route do you want your edge function to be invoked on? \nRout must begin with '/'\
+      and can be configured by editing your \`netlify.toml\` file.`,
+      default: '/test',
+    },
+  ])
+
+  const functionRegister = `\n\n[[edge_functions]]\nfunction = "${funcName}"\npath = "${funcPath}"`
+
+  fs.promises.appendFile('netlify.toml', functionRegister)
```

Comment:
We should be catching any errors from this before saying that the edge function was successfully registered.